serving/docker/deepspeed.Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM nvidia/cuda:11.3.0-cudnn8-devel-ubuntu20.04
```

Comment:
Let name it cu113.Dockerfile, this docker can be a generic GPU docker

Reply:
And the base docker image should be parameterized as well

Reply:
This is specialized for pytorch, transformers and deepspeed. For GPU generic docker, we should not install anything